src/other/Comms.js

```diff
@@ -374,25 +374,14 @@
             }
 
             var pathParts, queryParts;
-            if (!this._target && !this._query) {
-                // http://192.168.1.201:8002/WsEcl/res/query/hthor/quicktest/res/index.html
-                pathParts = this._pathname.split("/res/");
-                if (pathParts.length >= 2) {
-                    queryParts = pathParts[1].split("/");
-                    if (queryParts.length >= 3) {
-                        this.target(queryParts[1]);
-                        this.query(queryParts[2]);
-                    }
-                }
-            }
-            if (!this._target && !this._query) {
+            if (!this._target || !this._query) {
```

Comment:
Its the other way round - if either are null then I want to try an alternative parse.